test/chplcheck/MethodsAfterFields/MethodsAfterFields.precomp

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+$CHPL_HOME/tools/chplcheck/chplcheck MethodsAfterFields.chpl > MethodsAfterFields.message
```

Comment:
`precomp` files also get the name of the test (e.g. `MethodsAfterFields`) passed as argument `$1`. I suspect you could have something like:

```suggestion
$CHPL_HOME/tools/chplcheck/chplcheck $1.chpl > $1.message
```

The advantage is again being able to re-use these scripts for other linting rules.